AI.md

```diff
@@ -0,0 +1,43 @@
+## Guidelines relevant to AI-assisted contributions
+
+Tools using AI are increasingly used to produce text and code.  However ensuring
+the quality of AI-assisted output can be difficult, and contributors must be
+willing to adapt their workflows accordingly.  This section documents our
+expectations.  The guidelines also apply, or have a direct counterpart, with
+entirely human-produced content.
+
+1. We strive to evaluate proposed contributions based on their merit,
+   not their authors -- whether humans or programs.
+
+2. We ask that you have either written, or read and reviewed yourself,
+   every part of your contribution.  You are responsible for its quality.
+   (It is strongly recommended that you read a proposed contribution in full
+   even if you wrote all of it.)
+
+3. Reviewing AI-produced code can be more difficult than reviewing
+   human-written code and may require a different approach.
+   If a significant portion of your code, PR description, review comments or
+   messages on this repository have been entirely or almost-entirely
+   authored by an automated tool, we ask that you disclose that such work
+   was produced by automation rather than yourself.  Please mention which tool
+   was used and for what. (For example, "The testsuite additions were mostly
+   generated by Copilot.", or "This PR description was generated by ChatGPT from
+   a short summary.")  You should be adding acknowledgements anyway in the case
```

Comment:
"You should have an acknowledgement anyway if some part of your work came from another human contributor."